test/higher_order_ops/test_invoke_subgraph.py

```diff
@@ -695,6 +695,25 @@ def fn(x):
         res = opt_fn(x)
         self.assertEqual(ref, res)
 
+    def test_pending_unbacked(self):
+        @mark_compile_region
+        def gn(x):
+            u = x[0].item()
+            return x * u
```

Comment:
I think I am more curious about how you ran into this

Reply:
The deepseek model has a part that creates a mask and then indexes into the mask (creating the unbacked symint) but then the result has a fixed shape that is not dependent on the masked value.